inject-generator/src/main/java/io/avaje/inject/generator/GenericType.java

```diff
@@ -108,7 +104,9 @@ void addImports(ImportTypeMap importTypes) {
   }
 
   private static boolean includeInImports(String type) {
-    return type != null && !type.startsWith("java.lang.") && type.contains(".");
+    return type != null
+        && type.contains(".")
+        && (!type.startsWith("java.lang.") || Character.isLowerCase(type.charAt(10)));
```

Comment:
Not safe if length is less than 10

Reply:
it is though?  that code will only trigger if the fqn starts with `java.lang`

Reply:
effectively the code is
```java
  private static boolean includeInImports(String type) {
    if (type != null && type.contains(".")) {
      if (type.startsWith("java.lang.") && Character.isLowerCase(type.charAt(10))) {
        return false;
      }
      return true;
    }
    return false;
  }
```